manifests/ucl/config.pp

```diff
@@ -59,6 +59,12 @@
     default         => $key
   }
 
+  # key/value delimiter; only a space for UCL macros
+  $delemiter = $key ? {
+    /^\..*$/ => ' ',
```

Comment:
Please split that check out into a more speaking variable (`$isMacro` or something) and use that to determine the delimiter (note the spelling 😊) and the semicolon.